model/FieldType/DBComposite.php

```diff
@@ -223,11 +223,14 @@ public function hasField($field) {
 	 * @param string $field
 	 * @param mixed $value
 	 * @param bool $markChanged
+	 * @return $this
 	 */
 	public function setField($field, $value, $markChanged = true) {
+		$this->objCacheClear();
```

Comment:
Do have to clear all cache values here, and not simply the one for key `$field`?